lib/models/builder.js

```diff
@@ -12,15 +12,26 @@ var chalk       = require('chalk');
 var attemptNeverIndex = require('../utilities/attempt-never-index');
 var findBuildFile = require('../utilities/find-build-file');
 
-var FSMonitor = require('heimdalljs-fs-monitor');
-var monitor = new FSMonitor();
-monitor.start();
-
 var Sync = require('tree-sync');
 
 var signalsTrapped = false;
 var buildCount = 0;
 
+function _enableFSMonitorIfVizEnabled() {
+  var FSMonitor = require('heimdalljs-fs-monitor');
```

Comment:
@rwjblue ya, we started down that path. Then choose this trade-off. Not perfect, but reasonable balance maybe?